### PR TITLE
PMM-11323 added conf file and meta_metrics tests

### PR DESCRIPTION
**Pull request by vyarosh**

### CODE REVIEW

**Comment by vyarosh** [comment]
mysql conf file: https://github.com/Percona-Lab/pmm-submodules/actions/runs/4473983285

**Comment by vyarosh** [comment]
mysql conf file: https://github.com/Percona-Lab/pmm-submodules/actions/runs/4473983285
mongo telemetry: https://pmm.cd.percona.com/job/pmm2-ui-tests/31380/

File: tests/metrics/verifyMongoDB_test.js [hunk 1]
```diff
@@ -13,12 +13,22 @@ const mongo_test_user = {
...
+  replsetStatus: 'mongodb_collector_scrape_time_replset_status',
+};
+
 BeforeSuite(async ({ I }) => {
   const detectedPort = await I.verifyCommand('pmm-admin list | grep mongodb_node_1 | awk -F " " \'{print $3}\' | awk -F ":" \'{print $2}\'');
 
   connection.port = detectedPort;
-  await I.mongoConnect(connection);
-  await I.mongoAddUser(mongo_test_user.username, mongo_test_user.password);
+  // await I.mongoConnect(connection);
```
Comment (puneet0191) [inline tests/metrics/verifyMongoDB_test.js:30]: what exactly you doing here?

File: cli/tests/mysql-conf-file.spec.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      const serviceId = output.getStdOutLines().find((item) => item.includes('/service_id/')).trim()
+          .split(':').find((item) => item.includes('/service_id/')).trim();
+      await expect(async () => {
+        output = await cli.exec(`sudo pmm-admin list | grep _exporter | grep ${serviceId}`);
+        await output.outContains('Running');
+      }).toPass({
+        // Probe, wait 1s, probe, wait 2s, probe, wait 2s, probe, wait 2s, probe, ....
+        intervals: [1_000, 2_000, 2_000],
+        timeout: 10_000
+      });
```
Comment (puneet0191) [inline cli/tests/mysql-conf-file.spec.ts:36]: would you also like to add a check for ensuring metrics are also being collected?

**Review by puneet0191 — DISMISSED** [review]

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests-matrix/detail/pmm2-ui-tests-matrix/1878/pipeline/26/
Includes failed test which has no changes at all ... should not be impacted
https://pmm.cd.percona.com/job/pmm2-ui-tests/31435/testReport/(root)/MongoDB%20Collectors%20Parameters%20and%20Flags%20tests/PMM_T1280_Verify_that_pmm_admin_inventory_add_agent_mongodb_exporter_with___log_level_flag_adds_MongoDB_exporter_with_corresponding_log_levelPMM_T1282__PMM_T1284__PMM_T1291_Verify_that_pmm_admin_inventory_add_agent_node_exporter_with___log_level_flag_adds_Node_exporter_with_corresponding_log_level__not_ui_pipeline__mongodb_exporter__exporters/

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/job/pmm2-ui-tests-matrix/1887/

**Pull request merged**